From dd0247f89203b3b7c72b847ad2fea99be87dab6f Mon Sep 17 00:00:00 2001
From: Peng Fan <peng.fan@nxp.com>
Date: Wed, 12 Sep 2018 16:10:45 +0800
Subject: [PATCH 4595/5242] MLK-19554 ARM64: dts: 8qm domu: enable power key
 and correct dev_emmc

commit  bf95ab91714236bd1998eb4eda64a502d1320159 from
https://source.codeaurora.org/external/imx/linux-imx.git

Android will check /dev/input, so enable sc_pwrkey.
Correct dev_emmc path, because the address has changed use the physical
address.

Signed-off-by: Peng Fan <peng.fan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-mek-domu.dts     |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
index 03eba6a..ee91108 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
@@ -146,7 +146,7 @@
 					vendor {
 						compatible = "android,vendor";
 						/* emmc node which used if androidboot.storage_type=emmc */
-						dev_emmc = "/dev/block/platform/passthrough/15b010000.usdhc/by-name/vendor";
+						dev_emmc = "/dev/block/platform/passthrough/5b010000.usdhc/by-name/vendor";
 						type = "ext4";
 						mnt_flags = "ro,barrier=1,inode_readahead_blks=8";
 						fsmgr_flags = "wait,slotselect,avb";
@@ -553,7 +553,7 @@
 
 /delete-node/ &crypto;
 /delete-node/ &caam_sm;
-/delete-node/ &sc_pwrkey;
+/*/delete-node/ &sc_pwrkey;*/
 /delete-node/ &wdog;
 /delete-node/ &wu;
 
-- 
1.7.9.5

